int32 key
float64[] biases
WeightVector[] weight_vec
string activation
string aggregation
